puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/debian/release.groovy

```diff
@@ -0,0 +1,21 @@
+pipeline {
+    options {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Deploy') {
+            agent { label 'debian' }
+            steps {
+                parallel(
+                    "${major_version}/stretch": { push_debs_direct('stretch', major_version) },
```

Comment:
Expected string literal @ line 14, column 21 - no idea what it wants, but probably not a dynamic name of the step